windows-worker/README.md

```diff
@@ -0,0 +1,23 @@
+# zse-windows-worker
+buildbot-worker on windows 10
+
+## Prerequisites
```

Comment:
Which version of Windows 10 is needed and how should it be configured?

Reply:
We use the pro version at the moment. Added hint.